zones/linuxfests/linuxfests.org.zone

```diff
@@ -2,7 +2,11 @@ $ORIGIN linuxfests.org.
 @	900	IN	SOA	ns-1064.awsdns-05.org. awsdns-hostmaster.amazon.com. 1 7200 900 1209600 86400
 @	3600	IN	A	208.83.101.227
 @	3600	IN	AAAA	2607:ff38:2:6::e3
-@	3600	IN	MX	10 mail.linuxfests.org.
+@	3600	IN	MX	1   ASPMX.L.GOOGLE.COM.
```

Comment:
is it a requirement to have google at a weight of 1?  Having no lower (higher) weight available will make a migration a little bit harder.